lib/components/molecules/Form/DDFileInput/DDFileInput.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Label from '../../../atoms/Label';
+import Input from '../../../atoms/FileInput';
+import styles from './DDFileInput.style';
+import useOpenFileDialog from '../../../../core/utils/useOpenFileDialog';
+
+type Props = {
```

Comment:
Is this input type, use the original types and combine it with your component specific ones under types/index.js

Reply:
default export has been given to function.